gcloud/datastore/batch.py

```diff
@@ -21,7 +21,7 @@ class Local(object):
 
 from gcloud.datastore import _implicit_environ
 from gcloud.datastore import helpers
-from gcloud.datastore import datastore_v1_pb2 as datastore_pb
+from gcloud.datastore import _datastore_v1_pb2 as datastore_pb
```

Comment:
Maybe -- it's a protobuf, so I'm not sure if documenting it the way we currently do is the best option.

Can we have a separate section for this ? We don't have doc strings (since this will get blasted out from time to time), so it'll be pretty barebones...